src/OhmmsApp/RandomNumberControl.cpp

```diff
@@ -215,6 +217,7 @@ bool RandomNumberControl::put(xmlNodePtr cur)
     Random.init(pid,nprocs,mySeeds[pid],Offset+pid);
     app_log() << "  Random number offset = " << Offset
               <<  "  seeds = " << mySeeds[0] <<"-" << mySeeds[nprocs*omp_get_max_threads()] <<  std::endl;
+    app_log() << std::endl;
     if(nprocs<4)
```

Comment:
second nprocs<4

Reply:
This test seems more difficult to remove.  Without it, the list of random seeds keeps growing with number of processors.  Eventually it seems like it will be too large to be useful.

One the initial seed offset is known, all the rest of processor and thread seeds should be deterministic?

Reply:
I also remove completely. Bad on KNL with a large thread count.
Once a seed is given in the input. It is deterministic.